web/src/app/services/user.service.ts

```diff
@@ -114,4 +125,41 @@ export class UserService {
   formatUserSocial(response) {
     return response;
   }
+
+  saveUserSocialRecord(userData: UserSocial, provider: string) {
```

Comment:
Return type on all methods, I wasn't going to comment on each.

Reply:
@eddiejaoude  `saveUserSocialRecord` method has void return type. Should we add explicit return type for all the methods that have void | null return type

Reply:
yes please 👍 .

But don't mix `void` and `null` - its either got a return value or a `void` but not both